docs/data/material/pages.ts

```diff
@@ -328,8 +324,13 @@ const pages: MuiPage[] = [
   },
   {
     pathname: 'https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=sidenav',
-    title: 'Templates',
-    icon: standardNavIcons.ReaderIcon,
+    title: 'Templates store',
```

Comment:
Thought about adding "store" here to differentiate it from the "Templates" in the Getting started section. Once you enter the page, I know the title says "Free template", but I think it's worth the extra clarity here.